src/glog/logging.h.in

```diff
@@ -1752,15 +1767,15 @@ class GOOGLE_GLOG_DLL_DECL LogSink {
   // Returns the normal text output of the log message.
   // Can be useful to implement send().
   static std::string ToString(LogSeverity severity, const char* file, int line,
-                              const struct ::tm* tm_time,
+                              const LogMessageTime &logmsgtime,
                               const char* message, size_t message_len,
-                              int32 usecs);
+                              bool need_usec);
```

Comment:
Why is `usecs` now optional?

Reply:
Now `usecs `will be a part(member) of `LogMessageTime` and no need to be passed explicitly.

Reply:
Is this parameter still necessary?

Reply:
There are two versions of `send()` which call this function. One needs `usecs ` one doesn't. As I mentioned earlier If we can refactor the `send()` methods and retain only one, this parameter is not needed anymore.

Reply:
Got it. In that case, you can go ahead refactor the method to eliminate the parameter. The optional use of microseconds looks suspicious to me as it generates an inconsistency.

Reply:
Sure. Thanks!